### PR TITLE
GHOSTSW-8: Added ISS port.

### PR DESCRIPTION
**Pull request by sraaphorst**
Added the ISS port tab to the GHOST instrument editor.
We include this in a tab because we will have multiple tabs forthcoming for more information / configuration.

Screen Shot 2019-11-22 at 12 35 32 PM

### CODE REVIEW

**Comment by cquiroz** [comment]
Can the instrument be really in either port?
I'm not sure what's the use of port in the OT (this is not particular to GHOST) since the port is the in A&G and can be different than what's on the OT

**Comment by sraaphorst** [comment]
Yes, GHOST can be in either port. This was confirmed with Steve and was requested to be configurable via the OT. Setting the port in the OT, I believe, sets the position of the science area in the TPE so it affects AGS and user configuration of targets?

**Comment by cquiroz** [comment]
Ok, that makes sense. It is a pity that this setting isn't connected with the reality of where the instrument is at the time of the observation.
OTOH it has been that way for ever

File: bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala [hunk 1]
```diff
@@ -44,19 +45,37 @@ final class Ghost extends SPInstObsComp(GhostMixin.SP_TYPE) with PropertyProvide
...
     sc.putParameter(DefaultParameter.getInstance(Ghost.POS_ANGLE_PROP, getPosAngle))
+    sc.putParameter(DefaultParameter.getInstance(Ghost.PORT_PROP, getIssPort))
     sc
   }
+
+  /**
+   * ISS Port
+   */
+  private var port: IssPort = IssPort.DEFAULT
+  println("***** ISSPORT: " + port.displayValue())
```
Comment (cquiroz) [inline bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala]: println?

**Comment by tpolecat** [comment]
totally gridbag https://www.youtube.com/watch?v=UuLaxbFKAcc

**Comment by cquiroz** [comment]
`MigLayout` is the future of swing

File: bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala (hunk 1 above)
Comment (sraaphorst) [reply bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala]: LOL ooops. Thanks for catching that!

**Comment by cquiroz** [comment]
I have a special respect for anybody who can use `GridBagLayout`

**Pull request merged**